gossip/integration/gossip_test.sh

```diff
@@ -8,8 +8,9 @@ INTEGRATION_DIR="$( cd "$( dirname "$0" )" && pwd )"
 LOG_SERVER_COUNT=${1:-1}
 LOG_SIGNER_COUNT=${2:-1}
 HUB_SERVER_COUNT=${3:-1}
+HUB_CFG_TEMPLATE=${4}
```

Comment:
Is this meant to be part of this PR?

Reply:
Oops, no, that was a manual hack to allow testing -- removed.